.github/workflows/persistent_store_test.yml

```diff
@@ -12,20 +12,33 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
       - uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          target: thumbv7em-none-eabi
 
       - uses: actions-rs/toolchain@v1
         with:
           toolchain: nightly
-          override: true
+
+      - name: Build Persistent store library
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --manifest-path libraries/persistent_store/Cargo.toml --release --target=thumbv7em-none-eabi
 
       - name: Unit testing of Persistent store library (release mode)
         uses: actions-rs/cargo@v1
         with:
+          toolchain: nightly
```

Comment:
Is it necessary? The `rust-toolchain` file points to a specific nightly already. Do you need to override with a more recent one? Because this would have the effect that we unit-test the library with a different toolchain than the one we build against.

Reply:
I see 3 options:
- test only with frozen compiler
- test only with nightly
- test with both

I think they should be all equivalent (as long as the code compiles on both versions). First I wanted to do a matrix but it's not as simple because we can't name the rust-toolchain toolchain (in a maintainable way). But I can try again if we prefer to do everything (build + test) on both versions.